plugins/chat/assets/stylesheets/common/chat-drawer.scss

```diff
@@ -2,12 +2,39 @@ body.composer-open .chat-drawer-outlet-container {
   bottom: 11px; // prevent height of grippie from obscuring ...is typing indicator
 }
 
+.chat-drawer-resizer {
+  position: absolute;
+  top: -5px;
+  width: 15px;
+  height: 15px;
+}
```

Comment:
Could we also add a visual indicator to make this functionality more obvious?

Something like:

![resizeable-icon](https://user-images.githubusercontent.com/66427541/216616010-874f5d83-dc51-4a03-a955-725b8e7ee28a.jpg)

Reply:
Yes I tried something like this, but was never happy with the result, my plan is to defer this to chapoi once it's merged.

Reply:
I wanted to add a suggestion but I also tried and gave up :upside_down_face: